src/control/cmd/daos_server/auto_test.go

```diff
@@ -361,7 +361,7 @@ func TestDaosServer_Auto_confGen(t *testing.T) {
 				},
 			},
 			expCfg: control.MockServerCfg("ofi+psm2", tmpfsEngineCfgs).
-				WithNrHugePages(18432).
+				WithNrHugePages(19456).
```

Comment:
Magic numbers... is there a formula the test could use to calculate this?

Reply:
done